frontend/views/pages/GroupDashboard.vue

```diff
@@ -2,22 +2,28 @@
 page(pageTestName='dashboard' pageTestHeaderName='groupName' v-if='groupSettings.groupName')
   template(#title='') {{ groupSettings.groupName }}
 
-  start-inviting(v-if='groupMembersCount === 1')
+  welcome(v-if='!ourGroupProfile.incomeDetailsType')
 
-  proposals-widget
+  div(v-else)
 
-  page-section(title='This months overview')
-    overview
+    start-inviting(v-if='groupMembersCount === 1')
 
-  page-section(title='July Overview')
-    //- group-pledges-graph
-    progress-overview
+    div(v-if='canDisplayGraph')
+      page-section(title='This months overview' v-if='!showHistory')
+        span.support-history.button.is-outlined.is-small(@click='showHistory = !showHistory') Support history
+        overview
 
-  //- page-section(title='Support History')
-  //-   support-history
+      page-section(title='Support History' v-if='showHistory')
+        span.support-history.button.is-outlined.is-small(@click='showHistory = !showHistory') This month's overview
+        support-history
```

Comment:
My previous comment was marked as outdated, so it hasn't been addressed, so just leaving this here as a reminder that this whole section needs to be collapsed into a single separate widget.